partials/topnav.php

```diff
@@ -35,12 +37,10 @@
         'item_spacing' => 'preserve',
     );
     ?>
-    <nav class="nhsuk-header__navigation" id="header-navigation" role="navigation" aria-label="Primary navigation"
+    <nav class="jotw-header__topnav" id="header-navigation" role="navigation" aria-label="Primary navigation"
```

Comment:
I see jotw is used in a few new class names. I would suggest using 'hale' instead.

Reply:
Happy to change this.  Adam said something similar.

Reply:
Changed prefixes to `hale-`